Mars_QASpecFlow/StepDefinitions/LanguageStepDefinitions.cs

```diff
@@ -0,0 +1,143 @@
+using Mars_QASpecFlow.Pages;
+using Mars_QASpecFlow.Utilities;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using System;
+using TechTalk.SpecFlow;
+
+namespace Mars_QASpecFlow.StepDefinitions
+{
+    [Binding]
+    public class LanguageStepDefinitions : CommonDriver
+    {
+        LoginPage login = new LoginPage();
```

Comment:
Please make use of constructors, please see 
 https://www.tutorialspoint.com/csharp/csharp_classes.htm

https://www.knowledgehut.com/tutorials/csharp/csharp-constructors